experimental/packages/opentelemetry-sdk-metrics-base/src/export/MetricExporter.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AggregationTemporality } from './AggregationTemporality';
+import { MetricData } from './MetricData';
+
+
+// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#metricexporter
+
+// TODO should this just be an interface and exporters can implement their own shutdown?
+export abstract class MetricExporter {
```

Comment:
This file is just moved from the root src directory to src/export directory and re-indented with 2 spaces.

Reply:
We will most likely need to have two exporter interfaces for push and pull right? Doesn't affect this PR i'm just thinking

Reply:
The spec is said SDKs may choose the idiomatic design for their language. For both push/pull exporters, they need to talk to a MetricReader or be implemented as a MetricReader to collect metrics.

See: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#pull-metric-exporter